js/ui/grid_core/ui.grid_core.columns_controller.js

```diff
@@ -106,6 +106,7 @@ module.exports = {
              * @name dxDataGridOptions_columns
              * @publicName columns
              * @type Array<dxDataGridColumn>
+             * @fires dxDataGridOptions_onOptionChanged
```

Comment:
I think you should move this handler to GridBaseOptions_columns

Reply:
Fixed